ChangeLog

```diff
@@ -12,6 +12,13 @@ Version 17.4.0
     * Fix reduce builtin (#2925)
     * Fix issue with completing idents in the language server (#2917)
     * Improve performance when a large number of input targets are supplied (#2942)
+    * Virtualise the subrepo filesystem for remote execution to avoid downloading
+      the whole subrepo to plz-out (#2952, #2954, #2953, #2955, #2960, #2961,
+      #2962, #2963, #2966, #2968, #2965)
+    * Fix issue where we would double the architecture subrepo if it was explicityly
+      specified in a subrepo #2973
```

Comment:
is the convention to put brackets round the pr number?

Reply:
Yup fixed!